code/game/objects/effects/landmarks.dm

```diff
@@ -185,6 +185,14 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark)
 	name = "Psychologist"
 	icon_state = "Psychologist"
 
+/obj/effect/landmark/start/barber
+	name = "Barber"
+	icon_state = "Psychologist"
```

Comment:
bro

Reply:
oooh the way we do these are cool. we generate them in game and use rust-g to save them into a file. i forget where you do that though

Reply:
no actually we generate them entirely in game and the code for it is bugged due to a bug with icons in byond

Reply:
I need ti port my fix I made at skyrat

Reply:
someone got rid of the rustg_dmi_create_png thing it was cool